src/main/java/com/sss/garage/converter/split/SplitConverter.java

```diff
@@ -18,6 +18,7 @@ public SplitData convert(final Split source) {
         data.setName(source.getSplit());
         data.setLeague(getConversionService().convert(source.getLeague(), LeagueData.class));
         data.setDisplayText(data.getLeague().getDisplayText() + " - Split " + source.getSplit());
+        data.setDrivers(data.getDrivers());
```

Comment:
To nie za bardzo ma sens XD A jak zrobisz convert to będzie nieskończona pętla split konwertuje drivera ktory konwertuje split ktory konwertuje drivera itd. Trzeba się zastanowić kto ma być ownerem tej relacji. Myśle że bardziej split powinien mieć info o driverach, a jesli potrzebujemy tego na driverach to wyciagac to serwisem niz z data (do jakich splitow nalezy kierowca).

Reply:
Zajebałem się, miało być source XD Już poprawiłem